packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -609,21 +610,32 @@ contract Deploy is Deployer {
 
     /// @notice Deploy the OptimismPortal
     function deployOptimismPortal() public broadcast returns (address addr_) {
-        console.log("Deploying OptimismPortal implementation");
+        if (cfg.useInterop()) {
+            console.log("Deploying OptimismPortalInterop implementation");
 
-        OptimismPortal portal = new OptimismPortal{ salt: _implSalt() }();
+            OptimismPortalInterop portal = new OptimismPortalInterop{ salt: _implSalt() }();
 
-        save("OptimismPortal", address(portal));
-        console.log("OptimismPortal deployed at %s", address(portal));
+            addr_ = address(portal);
+
+            save("OptimismPortalInterop", addr_);
```

Comment:
Same feedback as for system config, we shouldn't differentiate in the naming of the 2 contracts, will reduce the amount of diff